docs/stdlib/bytes.rst

```diff
@@ -30,10 +30,10 @@ Bytes
       - Returns the number of bytes.
 
     * - :eql:func:`contains`
-      - Check if the byte sequence contains a subsequence.
+      - Checks if the byte sequence contains a subsequence.
 
     * - :eql:func:`find`
-      - Find the index of the first occurrence of a subsequence.
+      - Finds the index of the first occurrence in a subsequence.
```

Comment:
```suggestion
      - Finds the index of the first occurrence of a subsequence.
```

"in" should be reverted since we're finding the index *of* the subsequence in the overall sequence. The other change is good.